catalog/transformer/catalog-transformer-geojson-input/src/main/java/ddf/catalog/transformer/input/geojson/GeoJsonInputTransformer.java

```diff
@@ -96,37 +94,38 @@ public Metacard transform(InputStream input, String id)
             throw new CatalogTransformerException("Properties are required to create a Metacard.");
         }
 
-        String metacardTypeName = (String) properties.get(METACARD_TYPE_PROPERTY_KEY);
+        final String propertyTypeName = (String) properties.get(METACARD_TYPE_PROPERTY_KEY);
         MetacardImpl metacard = null;
 
-        if (metacardTypeName == null || metacardTypeName.isEmpty() || mTypeRegistry == null) {
+        if (propertyTypeName == null || propertyTypeName.isEmpty() || metacardTypes == null) {
             LOGGER.debug(
                     "MetacardType specified in input is null or empty.  Assuming default MetacardType");
             metacard = new MetacardImpl();
         } else {
-            QualifiedMetacardType metacardType = mTypeRegistry.lookup(metacardTypeName);
-            if (metacardType == null) {
-                String message =
-                        "MetacardType specified in input has not been registered with the system.  Cannot parse input.  MetacardType name: "
-                                + metacardTypeName;
-                LOGGER.warn(message);
-                throw new CatalogTransformerException(message);
-            }
-            LOGGER.debug("Found registered MetacardType: {}", metacardTypeName);
+            MetacardType metacardType = metacardTypes.stream()
+                    .filter(type -> type.getName()
+                            .equals(propertyTypeName))
```

Comment:
small nit, Could be `propertyTypeName.equals(...)` since you already ensured its not null.